dot/network/mdns.go

```diff
@@ -65,6 +65,7 @@ func (m *mdns) start() {
 	)
 	if err != nil {
 		log.Error("Failed to start mDNS discovery service", "err", err)
+		return
```

Comment:
well, if the discovery can fail and return a error, shall we just log it silently here ? is this the desired behavior ?

Reply:
I believe this is running inside a goroutine, if we want to check for errors we would need to create an error channel. mDNS isn't a critical component so I'm okay with just logging it

Reply:
@ryanchristo thoughts?